cucoslib/workers/stackaggregator.py

```diff
@@ -144,10 +144,21 @@ def _get_dependency_data (self, resolved, ecosystem):
             qstring += "as('version').in('has_version').as('package').select('version','package').by(valueMap());"
             payload = {'gremlin': qstring}
 
-            graph_req = requests.post(GREMLIN_SERVER_URL_REST, data=json.dumps(payload))
-            graph_resp = graph_req.json()
-
-            result.append(graph_resp["result"])
+            try:
+                graph_req = requests.post(GREMLIN_SERVER_URL_REST, data=json.dumps(payload))
+                if graph_req.status_code == 200:
+                    graph_resp = graph_req.json()
+
+                    if 'result' not in graph_resp:
```

Comment:
These few lines could be probably refactored and simplified, but the intent is clear here, so it's up to you.